site/gatsby-site/cypress/e2e/cite.cy.js

```diff
@@ -210,4 +210,23 @@ describe('Cite pages', () => {
       `Olsson, Catherine. (2014-08-14) Incident Number 10. in McGregor, S. (ed.) Artificial Intelligence Incident Database. Responsible AI Collaborative. Retrieved on ${date} from incidentdatabase.ai/cite/10.`
     );
   });
+
+  it('Should have OpenGraph meta tags', () => {
```

Comment:
You could use `cy.query` to fetch the actual incident title and make sure the proper value is there 🤔

Reply:
@cesarvarela done! Now it uses `cy.query`